dev/bin/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+ set -e
+
+ REPO_ROOT=$(git rev-parse --show-toplevel)
+
+ source ${REPO_ROOT}/dev/vendor/galoy-quickstart/bin/helpers.sh
+ source ${REPO_ROOT}/dev/.env
+
+ login_user "alice" "$ALICE_PHONE" "000000"
+ echo "alice logged in"
+ ALICE_TOKEN=$(read_value "alice")
+ receive_onchain
+ echo "alice funded"
+
+ login_user "alice" "$BOB_PHONE" "000000"
+ echo "bob logged in"
+ BOB_TOKEN=$(read_value "alice")
```

Comment:
Should this be reading "bob"

Reply:
We need bob's token in "alice" variable for receive_onchain, same as other comment.